agent-maven-3.5/Dockerfile.rhel7

```diff
@@ -20,7 +20,7 @@ ENV MAVEN_VERSION=3.5 \
 RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
     yum-config-manager --enable rhel-server-rhscl-8-rpms && \
     yum-config-manager --disable epel >/dev/null || : && \
-    INSTALL_PKGS="java-1.8.0-openjdk-devel.x86_64 rh-maven35*" && \
+    INSTALL_PKGS="java-1.8.0-openjdk-devel rh-maven35*" && \
```

Comment:
Am I correct in inferring that `java-1.8.0-openjdk-devel` == `java-1.8.0-openjdk-devel.x86_64` ?

Reply:
yes, you are correct. And in our case:
`java-1.8.0-openjdk-devel == java-1.8.0-openjdk-devel.ppc64le`